platforms/Windows/shared/swift.en-us.wxl

```diff
@@ -48,13 +47,21 @@
   <String Id="HelpCloseButton" Value="&amp;Close" />
   <String Id="InstallLicenseLinkText" Value="[WixBundleName] &lt;a href=&quot;#&quot;&gt;license terms&lt;/a&gt;." />
   <String Id="InstallAcceptCheckbox" Value="I &amp;agree to the license terms and conditions" />
-  <String Id="InstallOptionsButton" Value="&amp;Options" />
+  <String Id="InstallOptionsButton" Value="Customi&amp;ze" />
```

Comment:
Can you remind me what the `&amp;` does here. It looks weird putting it in the middle of the word and just want to make sure that's intentional.

Reply:
the shortcut character for the button. since there is a cancel on this form, "C" was taken, so pickled "z" instead.